lib/swagger/diff/diff.rb

```diff
@@ -80,12 +80,23 @@ def incompatible_request_params
         ret
       end
 
+      def new_child?(req, old)
+        idx = req.rindex('/')
+        return false unless idx
+        key = req[0..idx]
+        old.each do |param|
+          return false if param =~ /^#{key}/
```

Comment:
I think a `.starts_with?` would work here in place of a regex?  And `old.any? { |param| param.starts_with?(key) }` (removing the final `true` below) would be more idiomatic